### PR TITLE
page updates

### PR DESCRIPTION
**Pull request by Stephanie041996**
In this project i used a combination of grids and flex to achive a copy of the newyorktime page stracture and layout.

### CODE REVIEW

File: index.html
Comment (addod19) [inline index.html]: Kindly remove the commented code, use git history

File: index.html
Comment (addod19) [inline index.html]: The editors pick must use the `<aside>` tag, let us kindly fix it

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @Stephanie041996, great job on this project :+1: The page design is almost like the original, we have to do a little work on the design :smile: 

- Files not related to the project should be added to the .gitignore file like the `.vscode` file
- Kindly edit the PR. The PR must have a title and a short description of the project
- Kindly edit the README file, we must include a screenshot, tools used to build the project, authors social media accounts, instructions about how to clone the project, etc
- `Navigation` - The login element is not positioned correctly, kindly fix it as in the screenshot below:
login
- Let us kindly fix the position of the elements in the "more ins space" section
"Original"
more
"Clone"
more1
- Elements should be aligned with a combination of "float", "flex", and "grid". But we are missing the use of "Grid" in our CSS file
- The theory of inflation section must be aligned properly as in the original [site](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)

[Optional]
- It has a fixed footer (it sticks to the bottom part of the screen)
- The use of absolute positioning for the *editor's picks* and *subscribe* sections. The expected outcome can be achieved way more easily with the use of the `<section>` and `<aside>` tags which help keep the document flow
- Going forward, let us kindly improve our commit messages; for example; we can write "fix html errors" instead of "html fixed errors", "update css" instead of "css updates", etc. We can use `git amend` to fix this

Once these are resolved, kindly submit another code review request :smile: 
_Happy coding_ :woman_technologist: 
[Daniel Larbi Addo](linkedin.com/in/daniel-larbi-addo/)

File: README.md
Comment (christianotieno) [inline README.md]: ## Improve the README
- [x] The `README` looks okay but it can be bettered still. Kindly use this [template](https://github.com/microverseinc/readme-template) as an inspiration on areas you might want to improve upon and how to arrange items on this document.

File: index.html
Comment (christianotieno) [inline index.html]: ## Improve the Navigation menu 
- [x] The link image logo is not loading. I'd suggest you check how you are referencing the images in the `img` tag. This is what we currently have. 👇 
- [x] The social links should also be matching the original page and be centred on the page.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://stephanie041996.github.io/Newyork/)|
|:--:|:--:|
|header|header-clone|

File: index.html
Comment (christianotieno) [inline index.html]: ## Improve styling on the `editor's pick` section
- [x] Great job having implemented the `editor's pick` section using `aside` but it is wrongly placed. The images are not loading either. Kindly fix this.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://stephanie041996.github.io/Newyork/)|
|:--:|:--:|
|editor's-pick|editor's-pick-clone|

File: index.html
Comment (christianotieno) [inline index.html]: Improve styling on the `Theory of Inflation section
- [x] This section has its elements widely spaced and covering the entire page. It shouldn't. Instead, it should resemble that of the original page.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://stephanie041996.github.io/Newyork/)|
|:--:|:--:|
|theory-of-inflation|theory-of-inflation-clone|

File: index.html
Comment (christianotieno) [inline index.html:562]: ## Improve this section
- [x] Images in this section are not loading
- [x] The elements are widely spaced. they should not be covering the entire page.
- [ ] Implement the missing `Space and Astronomy` section as well.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://stephanie041996.github.io/Newyork/)|
|:--:|:--:|
|footer|footer-clone|

File: index.html
Comment (christianotieno) [inline index.html:725]: ## Implement the footer section
- [x] The footer section has not been implemented. Let's implement it.
- [x] Also the page need to have the fixed footer section in place as we.
- bottom-footer

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @Stephanie041996 👋 

Great job implementing the suggested pages. 👍 
The page still has some elements that need attention before we continue. 
The highlighted changes below 👇 touches on some missing areas in the footer that need to be implemented, images not loading and how some sections are placed on the page.

### Happy coding 💻 
#### Christian Otieno (CR)

File: .gitignore~d0bd33a6dd23e3137572d540e685d95bc4e483be
Comment (wathigo) [inline .gitignore~d0bd33a6dd23e3137572d540e685d95bc4e483be]: We already got an existing `.gitignore` file. Let's delete this one.

File: index.html
Comment (wathigo) [inline index.html:35]: The navigation bar is not well styles to look similar to the original space.
The text in login and subscribe buttons should be well aligned at the center. The icons still look much different together with Space & Cosmos heading.
Clone Page             |  Original Page
:-------------------------:|:-------------------------:
(Screenshot from 2021-02-18 18-30-08  | Screenshot from 2021-02-18 18-30-03

File: index.html
Comment (wathigo) [inline index.html:237]: This 'Theory of Inflation' section should not occupy the full width as mentioned by @christianotieno

File: index.html
Comment (wathigo) [inline index.html]: The footer does not look like the original one. It's totally different.
Clone Page             |  Original Page
:-------------------------:|:-------------------------:
Screenshot from 2021-02-18 18-43-19 | Screenshot from 2021-02-18 18-43-31

File: index.html
Comment (wathigo) [reply index.html:562]: The images and the paragraph below are not well aligned

**Review by wathigo — CHANGES_REQUESTED** [review]
## CHANGES REQUIRED :o: 
Great job implementing the requested changes.
We still have a few more issues left to address.
See the comments :point_down: 

**After making the changes, submit another request for review.**

**Happy Coding :computer: 
Simon Wathigo**

File: index.html
Comment (Stephanie041996) [reply index.html:35]: I am confused  what you mean I have worked on the nav bar alignment already, and the screenshot you are showing is of old version not current version with changed i made

File: index.html
Comment (Stephanie041996) [reply index.html:237]: I Have improved this and it is no longer occuply full section in new current version

image

File: index.html
Comment (Stephanie041996) [reply index.html]: It is the same I just am unable to style font  and line spacing to make look the same. please give suggestion of how to achive this.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @Stephanie041996  :wave:

__You've made laudable progress so far :clap: 
Most of the required changes have been made. 
You've worked so hard to get to this point, keep pushing :muscle: 
Let us look at some changes required before your project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] Your Readme is not well organized. Kindly use [this template](https://github.com/microverseinc/readme-template/blob/master/README.md) to ensure that your readme is structured with the appropriate markdown.
- [x] You can reduce the top and bottom margin of the top navigation bar.
- [x] You're having a horizontal scroll bar on your page. This might be caused by one of your containers. Kindly check and ensure that there's no horizontal scroll bar on your page.



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @Stephanie041996  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [x] Fix the subscribe section in your upper footer, learn the screenshots below to understand how you fix your version;

| Your clone | Original page |
| --- | --- |
|Screenshot from 2021-02-19 18-50-40|Screenshot from 2021-02-19 18-49-35|

:point_right: Could you see the difference :question: :question: :question: 

=> Make sure that you add a left border, and the four top options are bold and bigger than the bottom options

- [ ] Fix issues link in your readme file, currently it's broken
- [x] Use Proper markdow for links in your readme file, currently the links are written as if they are usual texts. Use this format `[text to display](url link)` => example: `[descholar-ceo](https://github.com/descholar-ceo)` will produce [descholar-ceo](https://github.com/descholar-ceo)
- [x] Make sure that you customize all of the sections from the readme file with your custom data. Don't leave those random titles `Setup, Install, Usage, Run tests, Deployment` empty.
___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Review by SpaYco — APPROVED** [review]
# Project Approved ✔️ 

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**
